src/modules/organizations/actions.js

```diff
@@ -0,0 +1,27 @@
+import {actionCreatorsFor} from 'redux-crud'
+import $ from 'jquery' // TODO(matthew): Is this needed at all?
+import * as displayErrorsActions from 'gModules/displayErrors/actions.js'
```

Comment:
if the symbol isn't anywhere in the file (which it doesn't seem to be), it isn't.